config/log4j.properties

```diff
@@ -10,3 +10,9 @@ log4j.logger.org.apache.kafka=ERROR, stdout
 log4j.logger.org.I0Itec.zkclient=ERROR, stdout
 log4j.additivity.kafka.server=false
 log4j.additivity.kafka.consumer.ZookeeperConsumerConnector=false
+
+log4j.appender.file=org.apache.log4j.DailyRollingFileAppender
```

Comment:
+1 RollingFileAppender is the safer bet.  If you really need the naming scheme provided by the DailyRollingFileAppender than probably the way to go is the RollingFileAppender + log4j extras that allow that to happen.